packages/animations/example/lib/container_transition.dart

```diff
@@ -0,0 +1,419 @@
+                width: 80,
+              ),
+            ),
+          ),
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text(
+                    'Title',
+                    style: Theme.of(context).textTheme.title,
+                  ),
+                  const Padding(
+                    padding: EdgeInsets.symmetric(vertical: 2.0),
+                  ),
+                  Text(
+                    subtitle,
+                    style: Theme.of(context).textTheme.caption,
+                  ),
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _ExampleSingleTile extends StatelessWidget {
+  const _ExampleSingleTile({this.openContainer});
+
+  final VoidCallback openContainer;
+
+  @override
+  Widget build(BuildContext context) {
+    const double height = 100.0;
+
+    return _InkWellOverlay(
+      openContainer: openContainer,
+      height: height,
+      child: Row(
+        children: <Widget>[
+          Container(
+            color: Colors.black38,
+            height: height,
+            width: height,
+            child: Center(
+              child: Image.asset(
+                'assets/placeholder_image.png',
+                width: 60,
+              ),
+            ),
+          ),
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.all(20.0),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Text(
+                    'Title',
+                    style: Theme.of(context).textTheme.subhead,
+                  ),
+                  const Padding(
+                    padding: EdgeInsets.symmetric(vertical: 4.0),
+                  ),
+                  Text(
+                      'Lorem ipsum dolor sit amet, consectetur '
+                      'adipiscing elit,',
+                      style: Theme.of(context).textTheme.caption),
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _InkWellOverlay extends StatelessWidget {
+  const _InkWellOverlay({
+    this.openContainer,
+    this.width,
+    this.height,
+    this.child,
+  });
+
+  final VoidCallback openContainer;
+  final double width;
+  final double height;
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: height,
+      width: width,
+      child: Stack(
+        children: <Widget>[
+          child,
+          InkWell(
+            splashColor: Colors.black38,
+            onTap: () {
+              Future<void>.delayed(
+                const Duration(milliseconds: 250),
```

Comment:
Why this delay?

Reply:
I just ran this on a real device and the delay feels strange.

Reply:
I added the delay because I saw that the Android version did have some sort of delay between tapping and having the new page show. The delay doesn't get it quite the same, but I was trying to achieve the effect where you can see the ink ripple. I could just remove this everywhere if it does feel too strange

Reply:
I would remove it.

Reply:
I took the delays off for now to make the app less complicated, and looking at it on a device, it does look pretty weird.